pkg/lib/userexport/model.go

```diff
@@ -24,14 +29,86 @@ var RequestSchema = validation.NewSimpleSchema(`
 		"format": {
 			"type": "string",
 			"enum": ["ndjson", "csv"]
+		},
+		"csv": {
+			"type": "object",
+			"properties": {
+				"fields": {
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"type": "object",
+						"properties": {
+							"pointer": {
+								"type": "string"
+							},
+							"field_name": {
+								"type": "string"
+							}
+						},
+						"required": [
+							"pointer"
+						]
+					}
+				}
+			}
 		}
 	},
 	"required": ["format"]
 }
 `)
 
+const DefaultCSVExportField = `
```

Comment:
Please make these changes

1. Change this variable to private.
2. Instead of making this a JSON document to be deserialized each time, initialize this to be a slice of the struct.

Reply:
updated to use slice of struct